README.md

```diff
@@ -22,7 +22,7 @@ Our default export contains all of our **ESLint rules**, and includes the follow
 
 #### Automated
 
-If you have `npx` available then you can run `npx install-peerdeps --dev @springernature/eslint-config` to add the `install-peerdeps` tool globally. This will automatically detect and use Yarn. 
+If you have `npx` available then you can run `npx install-peerdeps --dev @springernature/eslint-config` to install the dependencies automatically. This will automatically detect and use Yarn if available. 
 
```

Comment:
do we need the word 'automatically' twice in quick succession?
`...to install the dependencies automatically. This will detect and use Yarn if available.`

Reply:
dammit, no we don't.